docs/react-testing-library/api.md

```diff
@@ -252,34 +246,6 @@ expect(firstRender).toMatchDiffSnapshot(asFragment())
 
 ---
 
-## `cleanup`
```

Comment:
Let's keep this around, just modify it to explain why it exists and when you should use it (like if you're using a testing framework like `tape` or `ava`).